src/components/Chat/InputMessageV2.tsx

```diff
@@ -97,7 +118,7 @@ const styles = StyleSheet.create({
 });
 
 export interface InputMessageV2Props {
-  onSendButtonClicked: (message: string) => void;
+  onSendButtonClicked: (message: string, attachments: any) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onSendButtonClicked` function now accepts an additional parameter `attachments`. This is a breaking change if there are other parts of the code that use this function without providing the `attachments` argument. Make sure all usages of this function have been updated accordingly.



<!-- This is an auto-generated comment by OSS CodeRabbit -->